harness/tests/cli/test_experiment.py

```diff
@@ -7,8 +7,9 @@
 import determined
 import determined.cli
 from determined.common import constants
-from determined.common.api import bindings, certs
+from determined.common.api import _session, bindings, certs
```

Comment:
generally don't import anything with an `_`, except in an `__init__.py` file (for aliasing).  So in this case, `_session.Session` should be referenced as:

```python
from determined.common import constants, api
...
api.Session
```

The motivation is covered by our [code layout RFC](https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md).  Not all code that we have follows that rfc, unfortunately.  But all new code _should_ follow that rfc.

Reply:
Thanks! I hadn't noticed that `Session` was imported into `api`. Fixed.